services/wallet/thirdparty/wallet_types.go

```diff
@@ -0,0 +1,6 @@
+package thirdparty
+
+type WalletCommunityInfoProvider interface {
+	CommunityInfoProvider
+	FillCollectibleMetadata(collectible *FullCollectibleData) error
```

Comment:
Hmm if we're not gonna have separate Collectible-related and Token-related interfaces, we can simplify things and make `FillCollectibleMetadata` part of `CommunityInfoProvider` (`CommunityInfoProvider` is an interface to get wallet-relevant data from the community code, so this additional layer on top of it feels kind of redundant).
Anyway, not that important, leave it as is if you feel it's better.